index.html

```diff
@@ -35,6 +35,153 @@ <h1>Gabriel <br> Nwachukwu</h1>
+                <!-- this is a background image tag -->
+                <div class="bg-screen bg-img"></div>
+                <div class="project-name card-details">
+                    <h3>Project name goes here..</h3>
+                    <div class="languages">
+                        <div class="html language">Html/css</div>
+                        <div class="rails language">Ruby/Rails</div>
+                        <div class="js language">Javascript</div>
+                    </div>
+                    <a class="see-this-project" href="">
+                        <p class="text">see this projects</p>
+                        <img class="arrow" src="images/arrow.png" alt="arrow">
+                    </a>
+                </div>
+            </div>
+            <div class="card-container-grid">
+                <!-- this is a background image tag -->
+                <div class="bg-screen-1 bg-img"></div>
+                <div class="project-name-2 card-details">
+                    <h3>Project name goes here..</h3>
+                    <div class="languages">
+                        <div class="html language">Html/css</div>
+                        <div class="rails language">Ruby/Rails</div>
+                        <div class="js language">Javascript</div>
+                    </div>
+                    <a class="see-this-project" href="">
+                        <p class="text">see this projects</p>
+                        <img class="arrow" src="images/arrow.png" alt="arrow">
+                    </a>
+                </div>
+            </div>
+            <div class="card-container-grid">
+                <!-- this is a background image tag -->
+                <div class="bg-screen-2 bg-img"></div>
+                <div class="project-name-3 card-details">
+                    <h3>Project name goes here..</h3>
+                    <div class="languages">
+                        <div class="html language">Html/css</div>
+                        <div class="rails language">Ruby/Rails</div>
+                        <div class="js language">Javascript</div>
+                    </div>
+
+                    <a class="see-this-project" href="">
+                        <p class="text">see this projects</p>
+                        <img class="arrow" src="images/arrow.png" alt="arrow">
+                    </a>
+                </div>
+            </div>
+            <div class="card-container-grid">
+                <!-- this is a background image tag -->
+                <div class="bg-screen-3 bg-img"></div>
+                <div class="project-name-4 card-details">
+                    <h3>Project name goes here..</h3>
+                    <div class="languages">
+                        <div class="html language">Html/css</div>
+                        <div class="rails language">Ruby/Rails</div>
+                        <div class="js language">Javascript</div>
+                    </div>
```

Comment:
- [ ] It would be better if you use `ul` and `li` tags instead of `div` tags in order to make your code more Semantic 👏. Please change all your `languages` wrapper with `ul` and inner `div` tags with `li`.